src/main/java/core/basesyntax/CreationOfFigure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class CreationOfFigure extends FigureSupplier {
+    public static void main(String[] args) {
```

Comment:
Create field FigureSupplier and Figure array with some size. Initialize every element in this field according to the task, and in another loop call method `draw` from every single initialized figure.